build.sbt

```diff
@@ -23,7 +23,7 @@ lazy val root = (project in file(".")).settings(
   ),
   scalacOptions += "-deprecation"
 )
-(assembly / assemblyJarName) := "dr2-ingest-folder-opex-creator.jar"
+(assembly / assemblyJarName) := s"dr2-ingest-folder-opex-creator.jar"
```

Comment:
Was there supposed to be a variable within this string or is the `s` a mistake?

Reply:
I'm not sure why it's back, it's not supposed to be there. I've deleted it.